### DISCUSSION TITLE
Prevent modal from closing / minimum height

### DISCUSSION THREAD

**Discussion by timmyd87** [gorhom/react-native-bottom-sheet#1789] [Q&A]
Is there a way to stop  a user dismissing a modal by swiping down? 

If we look at apple maps, you cannot dismiss the modal completely, simply minimise it to the lowest snap point. Is there a way to achieve similar here?

**Comment by Tobim99** [comment]
enablePanDownToClose={false}

see https://github.com/gorhom/react-native-bottom-sheet/discussions/1061#discussioncomment-3390234
